### DISCUSSION TITLE
MSSqlServer support

### DISCUSSION THREAD

**Discussion by kunalghosh84** [pietermartin/sqlg#503] [General]
has removed MSSqlServer support from sqlg in version 3.0.0?

**Comment by pietermartin** [comment]
Yes, I can no longer justify the time to maintain the MSSqlServer dialect. I am unaware of anyone in the community using it.
That said, the code it still there, I am just not running the tests at the moment.

**Reply by ngskevinschmidt** [reply]
We do still use it, but have been stuck on an earlier version of Sqlg.  We are in the process of updating to Sqlg 3.x and so we are running into this now.  Can the code be brought back?

**Comment by pietermartin** [comment]
Hi, yes if you are using it then I'll put it back in again. If I recall you were also using `MySql`?
